blocklist-processing-engine/package.json

```diff
@@ -12,18 +12,21 @@
     "fix:prettier": "prettier --write \"**/*.ts\"",
     "lint": "yarn lint:eslint && yarn lint:prettier",
     "lint:eslint": "eslint \"**/*.ts\"",
-    "lint:prettier": "prettier --check \"**/*.ts\""
+    "lint:prettier": "prettier --check \"**/*.ts\"",
+    "test": "jest --config=jest.config.ts",
```

Comment:
We're using yarn but the package-lock.json exist in branch, can we remove?

Reply:
Currently the dockerfile's stil used the npm implementation. Will fix this in another PR